ci/run_tests.sh

```diff
@@ -62,20 +62,20 @@ run()
         if [ $? -ne 0 ]; then 
             echo "Error copying unit-test executable ${name} to ${BUILDBOT_PLATFORM} VM"
             exit 1
-        fi 	
+        fi
 
         # These tests should trigger an "In-address space security exception"
         # So they should fail, i.e. `exit=1`
         if [ "$1" = "to_fail" ]; then
             exit_status=0
             RESULT={{$(ssh -o "StrictHostKeyChecking no" -p ${SSHPORT} root@${SSHHOST} -t \
-	                   "LD_LIBRARY_PATH=/root:\$\{LD_LIBRARY_PATH\} ./$(basename ${name})")} && exit_status=1} || true
+                           "LD_LIBRARY_PATH=/root:\$\{LD_LIBRARY_PATH\} ./$(basename ${name})")} && exit_status=1} || true
```

Comment:
fixme :-)

Reply:
random tabs

Reply:
Actually, @jeremysinger , I was fixing the tabs that had accidentally crept in earlier. So I think this should stay.

Reply:
I'm a bit neutral here (tabs are bad; but I marginally preferred the previous level of indentation). However, one way or another, let's not mix reformatting up with code changes :) When it comes to squashing, let's break the reformatting stuff out into its own commit: we'll thank ourselves when we look back in the git history!

Reply:
Reverted in [a3ec991](https://github.com/capablevms/bdwgc/pull/16/commits/a3ec99146618e04d0041f82ff364393ec6d13761)